meditation_app/index.html

```diff
@@ -10,34 +10,36 @@
     <div class="app-wrapper">
         <section class="video">
```

Comment:
there is no point for using sections here. div is ok  plz check Usage notes here https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section